test/data/plugin/bundled/MANIFEST.MF

```diff
@@ -0,0 +1,393 @@
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters3.xml.txml.TXMLFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.tokenizer.LucenePortugueseTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.tokenizer.LuceneHindiTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters2.mozdtd.MozillaDTDFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.tokenizer.LuceneBulgarianTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org/omegat/filters2/
+Plugin-Name: File filters[bundle]
+Plugin-Category: filter
+Plugin-Description: Bundled filters for various file types
+
+Name: org.omegat.filters3.xml.wix.WiXFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.core.machinetranslators.IBMWatsonTranslate
+OmegaT-Plugin: machinetranslator
+
+Name: org.omegat.tokenizer.DefaultTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters3.xml.l10nmgr.L10nmgrFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.tokenizer.LuceneJapaneseTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters3.xml.helpandmanual.HelpAndManualFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.filters2.po.PoFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.core.machinetranslators.Google2Translate
+OmegaT-Plugin: machinetranslator
+
+Name: org.omegat.tokenizer.LuceneItalianTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters2.text.magento.MagentoFilter
+OmegaT-Plugin: filter
+
+Name: org/omegat/core/machinetranslators/
+Plugin-Name: MT connector[bundle]
+Plugin-Category: machinetranslator
+Plugin-Description: Bundled machine translator service connectors. There
+  are several connectors;  Apertium Translate / Belazar Translate / Deep
+ L Translate (only Pro user can use) /IBM Watson Translate / Microsoft T
+ ranslate V2 (obsolete, only user who has v2 key can use) / MyMemory Hum
+ an Translate / MyMemory Machine Translate / Google Translate v2 / Yande
+ x Cloud Translate (obsolete, only user who already has access key can u
+ se)
+
+Name: org.omegat.tokenizer.LuceneFinnishTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.tokenizer.LuceneHungarianTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters2.moodlephp.MoodlePHPFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.tokenizer.LuceneFrenchTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters3.xml.xmlspreadsheet.XMLSpreadsheetFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.tokenizer.LuceneRomanianTokenizer
+OmegaT-Plugin: tokenizer
+
+Name: org.omegat.filters2.rc.RcFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.filters2.text.bundles.ResourceBundleFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.filters2.text.dokuwiki.DokuWikiFilter
+OmegaT-Plugin: filter
+
+Name: org.omegat.core.machinetranslators.YandexCloudTranslate
+OmegaT-Plugin: machinetranslator
+
+Name: org/omegat/tokenizer/
+Plugin-Name: Tokenizers[bundle]
+Plugin-Category: tokenizer
+Plugin-Description: Bundled tokenizers plugin
+
+Name: org.omegat.tokenizer.LuceneIrishTokenizer
+OmegaT-Plugin: tokenizer
```

Comment:
A combination of  package section and specific class section, we can set values.

LuceneIrishTokenizer has category "tokenizer", then description "Bundled tokenizers plugin", with author "OmegaT team" that is defined in main attribute.